build.sh

```diff
@@ -101,11 +101,12 @@ FILE=.buildtools
 OS_NAME=`uname -a`
 if [ ! -f $FILE ]; then
   case "$OS_NAME" in
-    *Darwin*) tools/setup-buildtools-apple.sh $MAC_ARCH ;;
-    *Linux*)  [[ -z "$NOROOT" ]] && sudo tools/setup-buildtools.sh || echo "No root: skipping build tools installation." ;;
```

Comment:
Should `sudo` be added or removed here?

Reply:
There is no change for Linux. Only sudo is added for Mac.

Reply:
No change for Linux. Only adding sudo for Mac.